python/_brotli.cc

```diff
@@ -243,7 +343,9 @@ static PyMemberDef brotli_Compressor_members[] = {
 };
 
 static PyMethodDef brotli_Compressor_methods[] = {
-  {"compress", (PyCFunction)brotli_Compressor_compress, METH_VARARGS, brotli_Compressor_compress_doc},
+  {"process", (PyCFunction)brotli_Compressor_process, METH_VARARGS, brotli_Compressor_process_doc},
```

Comment:
FYI I went with the `process`, `flush`, `finish` naming to match the underly Brotli behavior.